### PR TITLE
Feat/allow previous subplot configs

### PR DESCRIPTION
**Pull request by geordle**
# Results of merging this

# What has been changed?
* remove auto opening plots
* Transforms old subplot config to new format if found, will remove in next major

### CODE REVIEW

File: test/setup/config/test_legacy_transforms.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from modules.setup.config import transform_subplot_config
+
+
+def test_transform_subplot_config_v0_7_3():
```
Comment (geordle) [inline test/setup/config/test_legacy_transforms.py:4]:
```suggestion
def test_transform_subplot_config_v0_7_4):
```

File: test/setup/config/test_legacy_transforms.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    result = transform_subplot_config(["indicator1", "indicator2"])
+    assert result == [["indicator1"], ["indicator2"]]
+
+
+def test_transform_subplot_config_v0_7_4():
```
Comment (geordle) [inline test/setup/config/test_legacy_transforms.py:9]:
```suggestion
def test_transform_subplot_config_v0_7_5():
```

File: test/setup/config/test_legacy_transforms.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from modules.setup.config import transform_subplot_config
+
+
+def test_transform_subplot_config_v0_7_4):
```
Comment (sheridavandenbent) [inline test/setup/config/test_legacy_transforms.py:4]:
```suggestion
def test_transform_subplot_config_v0_7_4():
```

**Review by sheridavandenbent — APPROVED** [review]
Fixed a small typo (bracket missing), rest is perfect!

**Pull request merged**